src/test/java/org/jitsi/meet/test/web/WebParticipantOptions.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright @ 2015 Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.meet.test.web;
+
+import org.jitsi.meet.test.base.*;
+
+import java.util.*;
+
+/**
+ * Holds web specific options.
+ */
+public class WebParticipantOptions
+    extends ParticipantOptions
+{
+    private static final String FAKE_AUDIO_PROP = "fakeStreamAudioFile";
+    private static final String FAKE_VIDEO_PROP = "fakeStreamVideoFile";
+    private static final String REMOTE_PROP = "isRemote";
+    private static final String BINARY_PROP = "binary";
+    private static final String VERSION_PROP = "version";
+
+    /**
+     * A prefix for global options (not per participant).
+     */
+    private static final String GLOBAL_PROP_PREFIX = "jitsi-meet";
+
+    /**
+     * Constructs new WebParticipantOptions, filling it with default values.
+     */
+    public WebParticipantOptions()
+    {
+        this.setProperty(FAKE_AUDIO_PROP, "resources/fakeAudioStream.wav");
```

Comment:
There will be a problem with setting default in the constructor when merging explicit ParticipantOptions on top of config loaded options (it will always override anything there). I think we should have a method somewhere (maybe in factory) which gets the default ParticipantOptions set and then all 3 options sets are being merge one on top of another. I'm looking into that now how that would looks like and if that make sense, as I'm not sure.

Reply:
But wait, these are defaults so if there are no settings in override this will stay. If there is something in those that will go on top of this, this is what we want. I think I don't get the problem.

Reply:
Yes, actually it will work this way. Forget about that